src/components/paragraph/link_teaser/index.tsx

```diff
@@ -0,0 +1,68 @@
+import { ParagraphLinkTeaser } from '@/types/paragraph'
+import { get } from 'lodash'
+
+function isRequestValid(paragraph: ParagraphLinkTeaser) {
+  return paragraph.field_link?.uri
+}
+
+const LinkTeaser = ({ paragraph, boldTitle, sectionHeader }): JSX.Element => {
+  if (!paragraph || !isRequestValid(paragraph)) return
+
+  const isFieldSpokes = paragraph.parent_field_name === 'field_spokes'
+  const fieldLinkOption = get(paragraph.field_link.options, ['target'], '')
+
+  if (isFieldSpokes || boldTitle) {
+    return (
+      <li key={paragraph.id} className="hub-page-link-list__item">
```

Comment:
@timcosgrove and @amponce we may need to call the recordEvent method here to track google analytics. Need to follow up.